evm/p2p/auth.py

```diff
@@ -206,18 +202,19 @@ def decode_ack_eip8(ciphertext, privkey):
 
     Returns the remote's ephemeral pubkey, nonce and protocol version.
     """
-    size = struct.unpack('>H', ciphertext[:2])[0] + 2
-    if len(ciphertext) != size:
-        raise ValueError("Invalid auth ack message size")
-    message = ecies.decrypt(ciphertext[2:size], privkey, shared_mac_data=ciphertext[:2])
+    # The length of the actual msg is stored in plaintext on the first two bytes.
+    encoded_size = ciphertext[:2]
+    (size,) = struct.unpack('>H', encoded_size)
```

Comment:
This `size` variable doesn't appear to be used anywhere...

Reply:
Indeed, removed